### DISCUSSION TITLE
docker: the configfile- 'app.ini' have wrong option in email config

### DISCUSSION THREAD

**Discussion by zoutao98** [gogs/gogs#6959] [中文讨论] [answered]
docker app.ini file is [mailer] with config email function.

But I found the source code in this repo is [email] to config email fucntion

**Comment by unknwon** [comment]
Hey, which "docker app.ini" you're referring to? I don't find a "app.ini" file under https://github.com/gogs/gogs/tree/main/docker 🤔

**Comment by zoutao98** [comment, marked as answer]
我在之前，按照仓库里的docker使用方法，拉取部署gogs/gogs的image
第一次启动容器之后访问gogs默认是安装页嘛，我就按照安装页的选项填写，完了之后发现邮件服务没成功，进到管理页面也没看到发送测试邮件的按钮；
我就开始排查问题嘛：我找到我自己本地容器卷，对应你文档里的这个位置
![图片](https://user-images.githubusercontent.com/46660687/173751780-e6ad2c4c-1681-4a46-a8d5-f847f7a51297.png)
这个目录里面不是有个app.ini文件
我对照你仓库里面的[app.ini](https://raw.githubusercontent.com/gogs/gogs/main/conf/app.ini)
发现邮件配置那一块和我本地容器卷里面的配置文件写的不一样，邮件配置是[mailer]；
然后我把它修改成[email],按照这个文件[app.ini](https://raw.githubusercontent.com/gogs/gogs/main/conf/app.ini)重新编写就可以使用邮件服务了

**Reply by yunkuangao** [reply]
Yes,it work.
